### ISSUE TITLE
No option to pick which card to use when multiple partial token keys will decrypt. 

### ISSUE THREAD

**Issue by TylerHelt0** [open-keychain/open-keychain#2639]
If a user attempts to decrypt a file or clipboard that is encrypted to more than one partial token (private key on smartcard) eg with standard syntax: 'gpg -r partial_token1 -r partial-token2 -e file' Openkeychain will only give the option to use the first smartcard used as recipient. User cannot pick another unless the order of the command is changed. E.g. 'gpg -r token2 -r token1 -e file' will prompt to use token 2 in openkeychain and will not allow using token1.

Tl;dr 
If you want to use a yubikey/whatever to store your private keys and want to use mutiple yubikeys for redundency, openpgp will only work with the first yubikey listed as a recipient in the encrypt operation.
